README.md

```diff
@@ -11,5 +11,19 @@ https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Resql/insert_regex.s
 
 https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Ruuter/POST/regex.yml
 
+```
+``` 
+
+### View list of Regex patterns added by user
+
+https://github.com/buerokratt/Data-Anonymizer/issues/60
+
+https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Resql/list_regex.sql
+
+```
+```
+
+https://github.com/buerokratt/Data-Anonymizer/blob/main/DSL.Ruuter/GET/regex.yml
+
 ```
 ```
```

Comment:
Please add a sample curl request.

Reply:
Done

Reply:
@turnerrainer please resolve this conversation